### ISSUE TITLE
Clicking on the back button in the arena screen does not work properly

### ISSUE THREAD

**Issue by Alayan-stk-2** [supertuxkart/stk-code#5226]
In most situations it will fail to trigger (it can still trigger sometimes if clicking in the right place / sequence). With keyboard navigation, it works fine ; so it must be due to something messing with mouse selection (I suspect it's the search bar).
